torch/quantization/fx/quantization_patterns.py

```diff
@@ -218,12 +218,33 @@ def __init__(self, quantizer: QuantizerCls, node: Node):
     def convert(self, quantizer: QuantizerCls, node: Node, load_arg: Callable,
                 debug: bool = False,
                 convert_custom_config_dict: Dict[str, Any] = None) -> Node:
+        # Supported combinations are:
+        # quant_type | activation (compute_type) | weight
+        #  static       quint8                      qint8
+        # tuple (activation_dtype, weight_dtype, compute_dtype)
```

Comment:
is L224 related to L221:L223?  It was a bit confusing to read, I might be missing something

Reply:
yeah this is a comment for supported_dtypes